charger/easee.go

```diff
@@ -392,9 +397,89 @@ func (c *Easee) Enable(enable bool) error {
 		action = easee.ChargeResume
 	}
 	uri := fmt.Sprintf("%s/chargers/%s/commands/%s", easee.API, c.charger, action)
-	_, err := c.Post(uri, request.JSONContent, nil)
 
-	return err
+	if err := c.postJSONAndWait(uri, true, nil); err != nil {
+		return err
+	}
+
+	if enable {
+		//reset currents after enable, as easee automatically resets to maxA
+		return c.MaxCurrent(int64(c.current))
+	}
+	return nil
+}
+
+// posts JSON to the Easee API endpoint and waits for the async response
+func (c *Easee) postJSONAndWait(uri string, isCommand bool, data io.ReadSeeker) error {
```

Comment:
Warum isCommand? Muss das auch für nicht-Commands aufgerufen werden?

Reply:
Ich denke isCommand ist äquivalent zu

    strings.Contains(uri, "/commands/")?

Reply:
Das Problem ist, daß Easee bei den Settings die Antworten in einem JSON Array verpackt, bei Commands jedoch ein einzelnes JSON Object zurücksendet. Ich habe aber bis jetzt noch keine Antwort gesehen, bei der im Array mehr als 1 Element steckte. Das API Design ist etwas schräg.
Ich werden den String Vergleich einbauen, dann wird zumindest die Methodensignatur schöner.